.env.example

```
@@ -0,0 +1,56 @@
+# Stripe Configuration
+STRIPE_SECRET_KEY=
+STRIPE_WEBHOOK_SECRET=
+
+# Frontend URL
+CORS_ORIGIN=http://localhost:3000
+
+# Social Providers
+GOOGLE_CLIENT_ID=
+GOOGLE_CLIENT_SECRET=
+
+STRIPE_SECRET_KEY=
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The STRIPE_SECRET_KEY key is duplicated

(DuplicatedKey)